requirements.txt

```diff
@@ -8,7 +8,7 @@ botocore==1.12.86
 certifi
 chardet==3.0.4
 click==6.7
-Cython==0.28.3
+Cython==0.27.3
```

Comment:
Any reason for this rolling back? Any potential side effect?

Reply:
We should monitor this... I got an error "numpy.ufunc has the wrong size, try recompiling." https://github.com/automl/auto-sklearn/issues/639#issuecomment-470441198

moving to an older cython and numpy fixed the issue

Reply:
Ok